cmd/util/cmd/compare-cadence-vm/cmd.go

```diff
@@ -96,9 +98,14 @@ func run(_ *cobra.Command, args []string) {
 	}
 	defer remoteClient.Close()
 
-	blockID, err := flow.HexStringToIdentifier(flagBlockID)
-	if err != nil {
-		log.Fatal().Err(err).Str("ID", flagBlockID).Msg("failed to parse block ID")
+	var blockIDs []flow.Identifier
+	for _, rawBlockID := range strings.Split(flagBlockID, ",") {
```

Comment:
What was previously a single ID seems to now be changed to a set / list of IDs. I would suggest to rename the `flagBlockID` accordingly and call it `flagBlockIDs`

Reply:
Good idea, improved in [dfef815](https://github.com/onflow/flow-go/pull/7955/commits/dfef815b5512e42dd90e33e7ac39946b3e8db45e)